scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -964,22 +968,39 @@ def get_string_period(_tag):
         can_direct_period = get_string_entry(BatchReductionEntry.CanDirectPeriod, row)
         output_name = get_string_entry(BatchReductionEntry.Output, row)
 
+        # If one of the periods is not null, then we should switch the view to multi-period view
+        if sample_scatter_period or sample_transmission_period or sample_direct_period or can_scatter_period or \
```

Comment:
It might be more readable to use `any` here:
```
if any((sample_scatter_period, sample_transmission_period, sample_direct_period, ...blah)):
```

Reply:
Changed this as you suggested as I think it is better.